versions/m-/mpg123.json

```
@@ -1,10 +1,15 @@
 {
   "versions": [
     {
-      "git-tree": "73f380749ae40814a07c7a0952e94db2970a9b85",
+      "git-tree": "c29e53dfebcdd51171128085c3a2e94272469e95",
       "version": "1.31.3",
       "port-version": 1
     },
+    {
+      "git-tree": "73f380749ae40814a07c7a0952e94db2970a9b85",
+      "version": "1.31.3",
+      "port-version": 2
+    },
```

```suggestion
    {
      "git-tree": "73f380749ae40814a07c7a0952e94db2970a9b85",
      "version": "1.31.3",
      "port-version": 2
    },
```
Please modify the port-version in `vcpkg.json`, and then use` ./vcpkg x-add-version {portname} --overwrite-version` to update the version.